tests/bluetooth/bsim_bt/bsim_test_mesh/_mesh_test.sh

```diff
@@ -37,7 +37,39 @@ function RunTest(){
     fi
 
     echo "Starting $testid as device #$idx"
-    Execute ./bs_${BOARD}_tests_bluetooth_bsim_bt_bsim_test_mesh_prj_conf \
+    Execute \
+      ./bs_${BOARD}_tests_bluetooth_bsim_bt_bsim_test_mesh_prj_conf \
+      -v=${verbosity_level} -s=$1 -d=$idx -RealEncryption=1 \
+      -testid=$testid
+    let idx=idx+1
+  done
+
+  count=$(expr $# - 1 + $extra_devs)
+
+  echo "Starting phy with $count devices"
+
+  Execute ./bs_2G4_phy_v1 -v=${verbosity_level} -s=$1 -D=$count
+
+  for process_id in $process_ids; do
+    wait $process_id || let "exit_code=$?"
+  done
+
+  if [ $exit_code != 0 ]; then
+    exit $exit_code #the last exit code != 0
+  fi
+
+  echo "#### Repeat test for Controller LOW_LAT variant ####"
+  idx=0
+  for testid in ${@:2} ; do
+    if Skip $testid; then
+      echo "Skipping $testid (device #$idx)"
+      let idx=idx+1
+      continue
+    fi
+
+    echo "Starting $testid as device #$idx"
+    Execute \
+      ./bs_${BOARD}_tests_bluetooth_bsim_bt_bsim_test_mesh_prj_low_lat_conf \
```

Comment:
Instead of this, it'd be better to just add
```sh
conf=${conf:-prj_conf}
Execute ./bs_${BOARD}_tests_bluetooth_bsim_bt_bsim_test_mesh_${conf} \
      -v=${verbosity_level} -s=$1 -d=$idx -RealEncryption=1 \
      -testid=$testid
```
to RunTest(), so we can call
```sh
RunTest mesh_transport_group transport_tx_group transport_rx_group
conf=prj_low_lat_conf
RunTest mesh_transport_group transport_tx_group transport_rx_group
```
in the test scripts, IMO.

The mesh tests generally shouldn't be adding test coverage for the controller, and this makes it impossible to avoid running on two controller variations, which would make debugging of mesh tests a bit of a pain. 

It's just these tests that would benefit from this dual mode testing, IMO:
- transport/unicast.sh
- transport/loopback_group.sh
- friendship/msg_mesh.sh

Reply:
Will make the suggested change to the RunTest.

> The mesh tests generally shouldn't be adding test coverage for the controller, and this makes it impossible to avoid running on two controller variations, which would make debugging of mesh tests a bit of a pain.

This is not coverage for the controller, it is two controller variants.